index.html

```diff
@@ -130,24 +130,24 @@ <h3 class="profession">I'am a Software Developer</h3>
       <li class="flex7">Ruby</li>
     </ul>
   </div>
-  <div class="button7"><button id="button7">See Project</button></div>
-</section>
-<section class="card8">
+  <div class="button7"><button id="button7" type="button">See Project</button></div>
+</div>
+<div class="card8">
   <div class="p8-title"><h1 id="p8-title">About me</h1></div>
```

Comment:
- [ ] For best sematic practices, your site should have only one main `h1` tag. As a result, it would be better to use an `h2` tag for the `About me` heading 👍